mlir/test/fusion/e2e/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+# Dependencies for the E2E test only
+set(FUSION_E2E_DEPENDS
+  FileCheck count not
+  rocmlir-opt
+  rocmlir-gen
+  rocmlir-driver
+  mlir-cpu-runner
+  mlir_rocm_runtime
+  conv-validation-wrappers
+  e2e_tests
+  fusion-e2e-tests
+)
+
+set(E2E_DIR ${CMAKE_CURRENT_BINARY_DIR})
+set(E2E_GEN_SCRIPT "${CMAKE_CURRENT_SOURCE_DIR}/generate-fusion-tests.py")
+set(CONFIG_DIR ${CMAKE_CURRENT_SOURCE_DIR})
+set(LOCAL_CONFIG_FILE_SRC ${CMAKE_CURRENT_SOURCE_DIR}/lit.local.cfg)
+set(LOCAL_CONFIG_FILE_BIN ${CMAKE_CURRENT_BINARY_DIR}/lit.local.cfg)
+
+# Add a command to copy lit.local.cfg
+add_custom_command(OUTPUT ${LOCAL_CONFIG_FILE_BIN}
+  COMMAND ${CMAKE_COMMAND} -E copy ${LOCAL_CONFIG_FILE_SRC} ${LOCAL_CONFIG_FILE_BIN}
+  COMMENT "Copying lit.local.cfg"
+)
+
+# Add a target to trigger E2E tests generation unconditionally
+add_custom_target(fusion-e2e-tests)
+add_custom_command(TARGET fusion-e2e-tests
+    COMMAND ${Python3_EXECUTABLE} ${E2E_GEN_SCRIPT} ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR}
+    COMMENT "Generating fusion E2E tests"
+  )
+
+# Add a new target to run the E2E test only
+add_lit_testsuite(check-fusion-e2e "Running fusion e2e tests"
```

Comment:
So I tried `ninja check-fusion-e2e` and I got the error:
/bin/sh: 1: /home/zhanglx/rocMLIR/build/bin/llvm-lit: not found
Can you check if this is the case on your end?
This the cmake command I used before running ninja
```
cmake -G Ninja -D CMAKE_BUILD_TYPE=RelWithDebInfo -DROCMLIR_DRIVER_ENABLED=1 -DROCMLIR_DRIVER_PR_E2E_TEST_ENABLED=1 -DROCMLIR_DRIVER_E2E_TEST_ENABLED=0 -DROCK_E2E_TEST_ENABLED=0 -DROCMLIR_DRIVER_TEST_GPU_VALIDATION=1 "-DLLVM_LIT_ARGS=-v --time-tests" -DCMAKE_EXPORT_COMPILE_COMMANDS=1 ..
```

Reply:
Hmm.  That's the whole reason I fiddled with LLVM_DEFAULT_EXTERNAL_LIT and LLVM_EXTERNAL_LIT.  I will see if I can reproduce the problem on a clean environment.

Reply:
Fixed this error.  Ultimately, it was because "add_subdirectory(fusion)" came before the setting of LLVM_LIT_TOOLS_DIR et al in mlir/test/CMakeLists.txt.